sage-ios/SAGE/SAGE/Announcement.swift

```diff
@@ -78,4 +78,10 @@ class Announcement: NSObject {
         }
         return propertyDict
     }
+}
+
+extension Announcement: NSCopying {
+    func copyWithZone(zone: NSZone) -> AnyObject {
```

Comment:
has to match exact function type so it has to return AnyObject
it implements a protocol and doesn't inherit, so no override